st2common/tests/unit/test_action_db_utils.py

```diff
@@ -132,6 +132,22 @@ def test_update_liveaction_status(self):
         self.assertDictEqual(newliveaction_db.context, context)
         self.assertEqual(newliveaction_db.end_timestamp, now)
 
+        # Update result in dict.
```

Comment:
Could we have a separate testcase. It is useful when reading why this test matters etc.